sycl/doc/design/OffloadDesign.md

```diff
@@ -208,9 +208,17 @@ The `sycl-post-link` tool is used after the device link is performed,
 applying any changes such as optimizations and code splitting before passing
 off to the `llvm-spirv` tool, which translates the LLVM-IR to SPIR-V.
 
-At this point, the SPIR-V binary is sent to the Ahead of Time compilation
-step to produce the final device binary or is wrapped before being linked into
-the final executable.
+|            Option                    |   Expected Behavior   |
```

Comment:
Are there any options that are need to be passed to ocloc for AOT.

Reply:
We have already added them in the text below this change.

Thanks